core/indexer/common.go

```diff
@@ -239,13 +239,29 @@ func canInterpretAsString(bytes []byte) bool {
 	return true
 }
 
-func serializeBulkMiniBlocks(hdrShardID uint32, bulkMbs []*Miniblock) bytes.Buffer {
+func serializeBulkMiniBlocks(
+	hdrShardID uint32,
+	bulkMbs []*Miniblock,
+	foundedObjMap func(hashes []string, index string) (map[string]bool, error),
```

Comment:
Should rename to something like `getAlreadyIndexedItems`. Just an idea, but `foundedObjMap` does not help to understand what it does

Reply:
good suggestion. renamed.